src/themes/simple/templates/Layout/JobBoardPage.ss

```diff
@@ -2,17 +2,26 @@
     <div class="container z1">
         <h2 class="title">$Title</h2>
         $Description
-        <hr class="mr-b-40">
-            <div class="row eqh gt40 fs-equalize-element">
 
+        <!-- filter widget -->
+        <div class="container filter-widget filter-style1 filterwidget1" id="filterwidget1" style="opacity: 1;">
+            <div class="filter-list">
+                <a class="active" data-filter="*">All</a>
+                <% loop $getJobLocations %>
+                    <a data-filter=".$URLSegment">$Title</a>
+                <% end_loop %>
+            </div>
+        </div>
+        <!-- /filter widget -->
+            <div class="row eqh gt40 fs-equalize-element grid">
                 <% loop $Children %>
-                    <div class="col-md-4" style="height: 253px;">
+                    <div class="col-md-4 element-item $loc2class($Location)" style="height: 253px;">
```

Comment:
Do we have classes defined somewhere for each location?  I'm trying to figure out what the loc2class thing is for.

Reply:
The `JobPage.php` class has that loc2class() method.

Reply:
I saw the method - was more curious about what the class is used for in css (this whole silverstripe world is a weird place)

Reply:
Oh true. Yea it shows/hides the div based on which location has been selected in the filter widget above it.